test/integration/options/posixExitCodes.spec.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+var helpers = require('../helpers');
+var runMocha = helpers.runMocha;
+
+describe('--posix-exit-codes', function () {
+  describe('when enabled, and with node options', function () {
+    var args = ['--no-warnings', '--posix-exit-codes'];
+
+    it('should exit with code 134 on SIGABRT', function (done) {
+      var fixture = 'signals-sigabrt.fixture.js';
+      runMocha(fixture, args, function postmortem(err, res) {
+        if (err) {
+          return done(err);
+        }
+        expect(res.code, 'to be', 134);
```

Comment:
expect(res.code, 'to be', signalOffset + os.constants.signals.SIGABRT);

Reply:
Resist hardcoded values outside 0, 1 and the empty string.

Reply:
btw, not sure this vs what expected processed, but signal numbers are only somewhat constant across various platforms.

https://stackoverflow.com/questions/44792204/why-are-several-signal-numbers-architecture-dependent-on-linux

https://unix.stackexchange.com/questions/155829/why-does-kill-l-not-list-signal-numbers-of-32-and-33

https://www-uxsup.csx.cam.ac.uk/courses/moved.Building/signals.pdf

Reply:
Good call.